agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/pipeline/TelemetryPipeline.java

```diff
@@ -132,6 +141,12 @@ private void onResponseBody(
 
     int responseCode = response.getStatusCode();
 
+    int newDailyQuotaExceeded = 402;
+    if (responseCode == newDailyQuotaExceeded) {
+      logger.warn("Stop sending telemetry data because new daily quota exceeded.");
+      this.stopSending = true;
+    }
```

Comment:
we'll want to start sending again when the quota gets next day's allotment, can we check the `retry-after` header from the 402 response? https://github.com/microsoft/Telemetry-Collection-Spec/blob/main/ApplicationInsights/sdk_behavior_breeze.md#402